src/constants/routes.ts

```diff
@@ -0,0 +1,7 @@
+export const ROUTES = {
+  ABOUT: "/about",
+  CALENDAR: "/calendar",
+  CONTACT: "/contact",
+  HOME: "/",
+  SPECIFIC_EVENT: "/calendar/events/:eventName",
+};
```

Comment:
The basic list of routes we will use all across the app. Notice how we put them in a separate file by themselves and import them where we need them elsewhere.